### DISCUSSION TITLE
More information in contact list

### DISCUSSION THREAD

**Discussion by tsr8** [FossifyOrg/General-Discussion#545] [Ideas]
I would like to see more information in contact list than the name.
It should be configurable.
What do you think?

**Comment by naveensingh** [comment]
What information would you like to see?

**Reply by tsr8** [reply]
Company at least. Maybe every non blank field should be visible.. or only fields selected i options.

**Reply by naveensingh** [reply]
I'm in favor of displaying phone number (optional) but only that. There's not enough space there for more fields.

**Reply by tsr8** [reply]
I have this work on old Xiaomi F1 phone and it was great.
If you want only a number so it should be configurable.
The same thing on call view (when someone call) I don't know who hi is without additional fields displayed.
